src/redux/types.js

```diff
@@ -0,0 +1,5 @@
+const ADD_BOOK = "redux/books/BOOK_ADDED";
+const REMOVE_BOOK = "redux/books/BOOK_REMOVED";
+const CHECK_STATUS = "redux/categories/BOOK_STATUS_CHECKED";
```

Comment:
- [Optional] Fantastic job for following duck patterns in defining action types 👍  but I think you don't have to define action type constants in a separate file. For a small project, it might be easier but when the project becomes larger it may be difficult to work with these constants in different files.